packages/edit-site/src/components/list/index.js

```diff
@@ -66,6 +79,15 @@ export default function List( { templateType } ) {
 					isDefaultOpen
 				/>
 			}
+			notices={
+				<div
+					className={ classnames( 'edit-site-list-snackbars', {
+						'is-navigation-open': isNavigationOpen,
```

Comment:
Should we move this logic to `InterfaceSkeleton` and I think we can avoid wrapper div.

Reply:
I changed this in aba861d, but feel free to revert.